src/ensembl/src/content/app/browser/drawer/drawer-views/DrawerBookmarks.tsx

```diff
@@ -36,10 +37,8 @@ export type DrawerBookmarksProps = {
 };
 
 const DrawerBookmarks = (props: DrawerBookmarksProps) => {
-  const limitedPreviouslyViewedObjects = props.previouslyViewedObjects.slice(
-    0,
-    props.previouslyViewedObjects.length - 20
-  );
+  const limitedPreviouslyViewedObjects =
+    props.previouslyViewedObjects.slice(20);
```

Comment:
This is the drawer code. Why is there a `slice(20)`? The deleted code did a different job.

Reply:
we always limit to 20 previously viewed object. previous code was working differently

Reply:
Sorry, I forgot the syntax of `Array.slice` for a moment there 😃 I thought this code is limiting previouslyViewedObjects to just 20 items; while in fact it is removing the first 20 items. All good.